float64[6] pose
---
int8 ok